components/ui/Tag.jsx

```diff
@@ -0,0 +1,12 @@
+function Tag(props) {
+  return (
+    <a
+      className={`h-fit rounded-4.75 bg-light-grey py-0.75 px-2.5 font-Neufile text-lg font-normal tracking-spacing-lg text-grey ${props.style}`}
```

Comment:
It looks good, only small thing to change, 
On tracking-spacing-lg I can't see the defined one in the config file . you should use the same as the one on figma   "letter-spacing: -0.01em "  and remember to define it in config.